frontend/tablet/src/components/Button.tsx

```diff
@@ -1,16 +1,22 @@
 import React from 'react'
-import { StyleSheet, Text } from 'react-native'
+import { StyleSheet } from 'react-native'
 import { TouchableOpacity } from 'react-native-gesture-handler'
 
-interface Props {
+import theme from './theme'
+
+import { Typography } from '.'
```

Comment:
You should not import from the same directory `index` file, it can create a dependency cycle. It should be:
```
import Typography from './Typography'
```